spec/xcpretty/formatters/simple_spec.rb

```diff
@@ -40,6 +40,11 @@ module XCPretty
         "> Compiling MainMenu.xib"
       end
 
+      it "formats compiling storyboard output" do
+        @formatter.format_compile_xib("Main.storyboard", 'path/to/file').should ==
+        "> Compiling Main.storyboard"
```

Comment:
Use 2 (not 0) spaces for indenting an expression spanning multiple lines.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.